docs/kratos/social-signin/95_account-linking.mdx

```diff
@@ -67,5 +93,13 @@ Automatic account linking can be a security risk. Consider this scenario:
 1. Your system, detecting duplicate accounts, prompts for account linking.
 1. Malicious actors link the accounts, gaining access to John's account.
 
-To prevent this users need to verify an additional credential before the accounts can be linked. In the scenario above, the
+To prevent this, users need to verify an additional credential before the accounts can be linked. In the scenario above, the
 malicious actors would be prompted to enter the password associated with the `jon@doe.com` identifier.
+
+### Hide irrelevant authentication methods
+
+Due to security considerations, it might be required to show all configured authentication methods to the user. However, this can
+be confusing for users who have multiple accounts with different authentication methods.
+
+To hide irrelevant authentication methods, you can use the [login hints](../../identities/sign-in/login-hint.mdx) feature. By
+enabling login hints, users are informed about the authentication methods available for the existing account.
```

Comment:
IMO a bit confusing what this means exactly. 
So when you enable login hints it hides the authentication method that the user selected in the first step (the method that is _not_ linked to their identity)?